synapse/storage/databases/main/event_federation.py

```diff
@@ -1122,6 +1144,21 @@ def _delete_old_forward_extrem_cache_txn(txn):
             _delete_old_forward_extrem_cache_txn,
         )
 
+    async def insert_backward_extremity(self, event_id: str, room_id: str) -> None:
+        def _insert_backward_extremity_txn(txn):
+            self.db_pool.simple_insert_txn(
+                txn,
+                table="event_backward_extremities",
```

Comment:
We should probably create our own table to keep track of this. Might be some unexpected consequences and `depth` assumptions trying to re-use this